docker/build

```diff
@@ -71,10 +71,10 @@ if [ ${#CANISTERS[@]} -eq 0 ]; then
 fi
 
 # Checking for dependencies
-if [[ ! "$(command -v ic-wasm)" || "$(ic-wasm --version)" != "ic-wasm 0.3.6" ]]; then
-  echo "could not find ic-wasm 0.3.6"
-  echo "ic-wasm version 0.3.6 is needed, please run the following command:"
-  echo "  cargo install ic-wasm --version 0.3.6"
+if [[ ! "$(command -v ic-wasm)" || "$(ic-wasm --version)" != "ic-wasm 0.8.5" ]]; then
+  echo "could not find ic-wasm 0.8.5"
+  echo "ic-wasm version 0.8.5 is needed, please run the following command:"
+  echo "  cargo install ic-wasm --version 0.8.5"
```

Comment:
If you can assume that `jq` is installed, you could do:
```
IC_WASM_VERSION="$(jq -r '.["ic-wasm"].version' dev-tools.json)"
```
and use the variable to save the repetition.  But this would be a drive-by cleanup and would maybe be better done in another PR.

Reply:
Indeed the goal of the PR was really just to bring up the tooling to the same version we have in other dapps. It can for sure be improved in another PR, not against!